api/v01/entities/vrp_input.rb

```diff
@@ -192,8 +192,9 @@ module VrpConfiguration
   end
 
   params :vrp_request_restitution do
-    optional(:geometry, type: Boolean, desc: 'Allow to return the MultiLineString of each route')
-    optional(:geometry_polyline, type: Boolean, desc: 'Encode the MultiLineString')
+    optional(:geometry, type: Array[Symbol], default: [], coerce_with: ->(value) { GeometryType.type_cast(value) },
```

Comment:
This should work ! But GitHub Actions will confirm.

Reply:
Since we expect Array[Symbol] it did not work and I had to remove those values. I will allow boolean also (https://github.com/Mapotempo/optimizer-api/pull/165/files#r606205605) and we should be able to use it.

Reply:
In fact it is tricky to use Boolean here : 

- if I use 
```suggestion
    optional(:geometry, type: [Array[Symbol], Boolean], default: [], coerce_with: ->(value) { GeometryType.type_cast(value) },
```
then it does not call coerce function when value is false. I think I need to use current line and, when value is true, return like [:to_fill]. In this case I can use values but can not output _to_fill_ in the documentation.